src/Creitive/Breadcrumbs/Breadcrumbs.php

```diff
@@ -411,11 +412,15 @@ protected function renderCrumb($name, $href, $isLast = false)
 
 		if (!$isLast)
 		{
-			return "<li><a href=\"{$href}\">{$name}</a>{$divider}</li>";
+			return '<li itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem" >'
+				. "<a itemprop=\"item\" href=\"{$href}\"><span itemprop=\"name\">{$name}</span></a>"
+				. "<meta itemprop=\"position\" content=\"{$position}\" />{$divider}</li>";
```

Comment:
I don't know much about microdata, so is the `position` element required for this to work correctly? This continues on my previous comment about setting `$position` to `null` by default.

Maybe just return the old rendering (the removed line) if `$position` is `null`, what do you think about that?